agb/src/lib.rs

```diff
@@ -161,7 +162,7 @@ pub use agb_macros::entry;
 pub use agb_sound_converter::include_wav;
 
 extern crate alloc;
-mod agb_alloc;
+pub mod agb_alloc;
```

Comment:
I'd be tempted, rather than making `agb_alloc` a `pub` module instead rexporting it to the top level. I think `agb::IWRAM_ALLOC` works prefectly well :).